recipes/benchmark/all/conanfile.py

```diff
@@ -35,6 +35,20 @@ class BenchmarkConan(ConanFile):
         "enable_libpfm": False,
     }
 
+    @property
+    def _min_cppstd(self):
+        return 14
```

Comment:
```suggestion
        return 14 if Version(self.version) >= "1.8.5" else 11
```
Shouldn't C++11 be set for the older versions?